timemachine/fe/estimator.py

```diff
@@ -130,52 +229,31 @@ def simulate(
         barostat.seed = np.random.randint(np.iinfo(np.int32).max)
 
     intg_impl = integrator.impl()
-    baro_impl = barostat.impl(all_impls)
+    # technically we need to only pass in the nonbonded impl
+    barostat_impl = barostat.impl(all_impls)
     # context components: positions, velocities, box, integrator, energy fxns
-    ctxt = custom_ops.Context(
-        x0,
-        v0,
-        box,
-        intg_impl,
-        all_impls,
-        barostat=baro_impl,
-    )
-    base_interval = baro_impl.get_interval()
-    # Use an interval of 5 for equilibration
-    baro_impl.set_interval(5)
+    ctxt = custom_ops.Context(x0, v0, box, intg_impl, all_impls, barostat_impl)
 
     # equilibration
     equil_schedule = np.ones(equil_steps) * lamb
     ctxt.multiple_steps(equil_schedule)
 
-    baro_impl.set_interval(base_interval)
+    # (ytz): intentionally hard-coded, I'd rather the end-user *not*
+    # muck with this unless they have a good reason to.
+    barostat_impl.set_interval(25)
```

Comment:
Clarification: does this change the barostat frequency for production and/or equilibration?

(Possibly a more explicit / visible way to hard code this might be to put this in signatures of `simulate(..., barostat_interval=25)` and `equilibrate(..., barostat_interval=25)`, and raise value error if != 25... Side note: function `equilibrate` is used in RBFEModel.equilibrate_edges, but maybe also should be used at https://github.com/proteneer/timemachine/blob/20c83a8c5e6b334bdd50ba5100a41e1b1623fd6a/timemachine/fe/estimator.py#L237-L239?)

Reply:
Currently everywhere that is using 25 steps. 

In all likelihood that `equilibrate` function should disappear. I implemented the behavior in `equilibrate_edges` incorrectly and should probably be replaced with `minimizer.equilibrate_host`. I imagine this will also change when we start using Single Topology again and didn't want to muddy this PR with changes outside the scope of merging the two files.

I hesitate to raise a value error in this code as it runs on the servers. If someone changed the value and ran it in parallel could end up wasting a lot of resources to raise an exception that would occur for all simulations.